qiskit_ibm/ibm_provider.py

```diff
@@ -60,20 +59,13 @@ class IBMProvider(Provider):
         from qiskit_ibm import IBMProvider
         IBMProvider.save_account(token=<INSERT_IBM_QUANTUM_TOKEN>)
 
-    The open access provider (`ibm-q/open/main`) is the default provider, but you can overwrite
-    this default using the `hub`, `group`, and `project` keywords in `save_account()`.
-    Once credentials are saved you can simply instantiate the provider like below to load the
-    saved account and default provider::
+    You can set the default project using the `hub`, `group`, and `project` keywords
+    in `save_account()`. Once credentials are saved you can simply instantiate the
+    provider like below to load the saved account and default project::
```

Comment:
Since h/g/p is selection is now done under the cover now, I think it's worth a few words explaining how the selection is done if not explicitly specified.